framework/tests/test_ui.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from framework.forms.artist_form import ArtistForm
+from framework.forms.search_form import SearchForm
+from framework.forms.spotify_home_form import SpotifyHomeForm
+from selenium.webdriver.common.by import By
+from browser.py_quality_services import PyQualityServices
+
+class TestSpotifyUI:
+    __spotify_home_form: SpotifyHomeForm = SpotifyHomeForm()
+    __artist_form: ArtistForm = ArtistForm()
+    __search_form: SearchForm = SearchForm()
+
+
+    @pytest.mark.parametrize("artist, song", [
+        ("Drake", "One Dance"),
+        ("The Beatles", "Here Comes The Sun - Remastered 2009")
+    ])
+
+
+
```

Comment:
please remove redundant empty lines across this class and add an empty line at the end of file